apps/rush-lib/src/schemas/command-line.schema.json

```diff
@@ -187,6 +187,28 @@
               "items": {
                 "type": "string"
               }
+            },
+            "watchOptions": {
+              "title": "Watch Options",
+              "description": "Controls the file watching behavior of this command. If not specified, this command does not watch files.",
+              "type": "object",
+              "additionalProperties": false,
+              "required": ["watchForChanges", "watchPhases"],
+              "properties": {
+                "watchForChanges": {
```

Comment:
do you really need a watchForChanges when you already define a watchOptions: {} ?

Reply:
I like having an explicit "this is a watch command" setting.